frontend/app_flowy/packages/flowy_editor/example/lib/main.dart

```diff
@@ -205,4 +216,34 @@ class _MyHomePageState extends State<MyHomePage> {
       child: TextField(),
     );
   }
+
+  Widget _buildFlowyEditorWithBigDocument() {
+    return FutureBuilder<String>(
+      future: rootBundle.loadString('assets/big_document.json'),
+      builder: (context, snapshot) {
+        if (!snapshot.hasData) {
```

Comment:
I think we should write the normal path through the code first; then write the unusual cases.

if (snapshot.hasData) {
// normal path
} else {
}

Reply:
I think we should put the code which's body is shorter before.

Reply:
Well, extracting the else condition into a function call would be better.

Reply:
Also, I found the e-version of the code complete. https://nikola-breznjak.com/blog/books/programming/code-complete-2-steve-mcconnell-using-conditionals/

Reply:
Got it.